web/src/app/workplace/tasklist/tasklist.component.scss

```diff
@@ -16,10 +16,9 @@
 }
 
 ul {
-  max-height: 90vh;
+  max-height: 88.5vh;
```

Comment:
why? if you are playing with vh is probably because you need to have a look to site.scss
taskana-workbasket-information,taskana-task-details, taskana-workbasket-access-items, taskana-workbaskets-distribution-targets, taskana-monitor-tasks,
taskana-monitor-workbaskets, taskana-monitor-classification-tasks, taskana-access-items-management, taskana-classification-details,taskana-workbasket-details {
    & .panel{
        border: none;
        box-shadow: none;
        margin-bottom: 0px;
        &> .panel-body {
            height: calc(100vh - 108px);
            max-height: calc(100vh - 108px);
            overflow-y: auto;
        }
    }
}